README.md

```diff
@@ -26,15 +26,15 @@ it into it's final memory location.
 
 This library allows you to do in-place initialization safely.
 
-### Nightly Needed for `alloc` and `std` features
+### Nightly Needed for `alloc` feature
 
-This library requires unstable features when the `alloc` or `std` features are enabled and thus
-can only be used with a nightly compiler. The internally used features are:
-- `allocator_api`
-- `get_mut_unchecked`
+This library requires the `allocator_api` unstable feature when the `alloc` feature
+is enabled and thus this feature can only be used with a nightly compiler.  The feature
+is enabled by default, but it is possible to enable `std` only instead to use the crate
+on stable compilers.
```

Comment:
I don't understand what you're trying to say with this sentence.

Reply:
Reworded, the idea was to say "you can disable alloc, but you need to keep std to get a usable crate on the stable compilers"

Reply:
Hmm, I don't think that std is required for the crate to be usable. One could have an entirely custom `alloc` that is also infallible.

Reply:
The problem is not getting the allocator from std, it's that Box and to a lesser extent Arc are too magic to reimplement them. You can't write your own Box without unstable features and intrinsics, at which point you are back using nightly features.

So I don't think it's possible to use pinned-init on stable *and* without std.